# Contains 2 discrete values 

# Represents id of the current interaction
int64 id 

# Represents size of the object (small or big)
int64 object_size
